Katana/Store.swift

```diff
@@ -339,6 +359,34 @@ open class Store<S: State, D: SideEffectDependencyContainer>: PartialStore<S> {
   override public func dispatch<T: SideEffect>(_ dispatchable: T) -> Promise<T.ReturnValue> {
     return self.enqueueSideEffect(dispatchable)
   }
```

Comment:
do we still need this? or can we drop it? same for context function.

Reply:
I tested without the `SideEffect` in `Ramen` and `RamenDemo` and it works! :D I'm removing them in the next commit to keep the API as simple as possible! ;)